src/contracts/relay-adapt/__tests__/relay-adapt.test.ts

```diff
@@ -1048,7 +1046,9 @@ describe('relay-adapt', function test() {
     expect(relayAdaptAddressBalance).to.equal(0n);
 
     const callResultError = RelayAdaptV2Contract.getRelayAdaptCallError(txReceipt.logs);
-    expect(callResultError).to.equal('Unknown Relay Adapt error.');
+    expect(callResultError).to.equal(
+      'Unknown Relay Adapt error: No utf8 string parsed from revert reason.',
+    );
```

Comment:
The error message was updated in the implementation, so updating it in tests too.